var/spack/repos/builtin/packages/cp2k/package.py

```diff
@@ -80,32 +80,57 @@ def install(self, spec, prefix):
             optflags = {
                 'gcc': ['-O2',
                         '-ffast-math',
-                        '-ffree-form',
-                        '-ffree-line-length-none',
                         '-ftree-vectorize',
                         '-funroll-loops',
                         '-mtune=native'],
                 'intel': ['-O2',
                           '-pc64',
                           '-unroll',
-                          '-heap-arrays 64']
+                          '-xHost']
```

Comment:
On our system, we add the following to compilers.yaml:
```yaml
    flags:
      cflags: -axCORE-AVX512,CORE-AVX2,AVX
      cxxflags: -axCORE-AVX512,CORE-AVX2,AVX
      fflags: -axCORE-AVX512,CORE-AVX2,AVX
```
This lets us optimize for all 3 instruction sets. If you only care about your current host, you may want to add `-xHost` there.